src/script/components/Modals/LegalHoldModal/LegalHoldModal.state.ts

```diff
@@ -137,12 +137,14 @@ const useLegalHoldModalState = create<LegalHoldModalState>((set, get) => ({
           .join('')
       : '';
 
+    const isOpen = showLoading ? showLoading : !!formattedFingerprint;
```

Comment:
```suggestion
    const isOpen = showLoading || !!formattedFingerprint;
```

Or is there something I'm missing?